include/sys/uzfs_zvol.h

```diff
@@ -47,6 +47,25 @@ typedef struct metaobj_blk_offset {
 	uint64_t m_offset;
 } metaobj_blk_offset_t;
 
+/*
+ * rebuild related information for zvol
+ */
+typedef struct zvol_rebuild_data {
+	/* mutex to synchronize io tree operation */
+	kmutex_t io_tree_mtx;
+	uint64_t rebuild_bytes;
+	avl_tree_t *incoming_io_tree;   /* incoming io tree */
+} zvol_rebuild_data_t;
+
+/*
+ * zvol state
+ */
+typedef enum zvol_status {
+	ZVOL_STATUS_HEALTHY,		/* zvol has latest data */
+	ZVOL_STATUS_DEGRADED,		/* zvol is missing some data */
+	ZVOL_STATUS_REBUILDING		/* zvol is in rebuilding state */
```

Comment:
REBUILDING state should be part of zvol_rebuild_status rather than zvol_status

Reply:
ok